### PR TITLE
Issue/2547 - Add affiliate referral to WooCommerce 3.3 preview order modal window

### PR DESCRIPTION
**Pull request by tubiz**
Issue: #2547

### CODE REVIEW

File: includes/integrations/class-woocommerce.php [hunk 1]
```diff
@@ -960,6 +964,41 @@ public function render_orders_referral_column( $column_name, $order_id ) {
 		}
 
 	}
+
+	public function order_preview_get_referral( $order_details, $order ) {
```
Comment (pippinsplugins) [inline includes/integrations/class-woocommerce.php]: This will need PHPdoc

File: includes/integrations/class-woocommerce.php [hunk 2]
```diff
@@ -960,6 +964,41 @@ public function render_orders_referral_column( $column_name, $order_id ) {
...
+
+		$referral_html .= '</div>';
+
+		$order_details['referral'] = $referral_html;
+
+		return $order_details;
+
+	}
+
+	public function render_order_preview_referral() {
```
Comment (pippinsplugins) [inline includes/integrations/class-woocommerce.php]: This will need PHPdoc

Comment (tubiz) [reply includes/integrations/class-woocommerce.php]: Added.

File: includes/integrations/class-woocommerce.php (hunk 1 above)
Comment (tubiz) [reply includes/integrations/class-woocommerce.php]: Added.

**Pull request merged**